client/data/emails/use-create-titan-mailbox-mutation.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+import { useMutation } from 'react-query';
+import wpcom from 'calypso/lib/wp';
+
+/**
+ * Creates a mailbox for a Professional Email (Titan) account
+ *
+ * @param {string} domainName The domain name of the mailbox
+ * @param name {string} The name of the user associated with the mailbox
+ * @param {string} mailboxName The mailbox name
+ * @param password {string} The password to the mailbox
```

Comment:
I'm a bit lost: sometimes the type is listed first and other times it is listed after the parameter name.